README.md

```diff
@@ -0,0 +1,355 @@
+@code {
+    
+    // The `_news` field will hold all our news. We will have a look at the `News`
+    // class in just a moment.
+    private List<News>? _news;
+
+    // `OnInitializedAsync()` gets called when the website is loaded. Our data
+    // retrieval logic has to be placed here.
+    protected override async Task OnInitializedAsync()
+    {
+        // First, we need to create a `MongoClient` which is what we use to
+        // connect to our cluster.
+        // The only argument we need to pass on is the connection string you
+        // retrieved from Atlas. Make sure to substitute the password with
+        // the actual password.
+        var connectionString = Environment.GetEnvironmentVariable("CONNECTION_STRING");
+        var mongoClient = new MongoClient(connectionString);
+        // Using the `mongoCLient` we can now access the database.
+        var cryptoNewsDatabase = mongoClient.GetDatabase("crypto-news-database");
+        // Having a handle to the database we can furthermore get the collection data.
+        // Note that this is a generic function that takes `News` as it's parameter
+        // to define who the documents in this collection look like.
+        var newsCollection = cryptoNewsDatabase.GetCollection<News>("news");
+        // Having access to the collection, we issue a `Find` call to find all documents.
+        // A `Find` takes a filter as an argument. This filter is written as a `BsonDocument`.
+        // Remember, `BSON` is really just a (binary) JSON.
+        // Since we don't want to filter anything and get all the news, we pass along an
+        // empty / new `BsonDocument`. The result is then transformed into a list with `ToListAsync()`.
+        _news = await newsCollection.Find(new BsonDocument()).ToListAsync();
+        // And that's it! It's as easy as that using the driver to access the data
+        // in your MongoDB Atlas cluster.
+    }
+
+}
+```
+
+In the above, you've noticed the `News` class which still needs to be created.
+In the `Data` folder, add a new C# class, call it `News` and use the following code.
+
+```csharp
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace CryptoNewsApp.Data
+{
+    public class News
+    {
+        // The attribute `BsonId` signals the MongoDB driver that this field 
+        // should used to map the `_id` from the Atlas document.
+        // Remember to use the type `ObjectId` here as well.
+        [BsonId] public ObjectId Id { get; set; }
+
+        // The two other fields in each news are `title` and `date`.
+        // Since the C# coding style differs from the Atlas naming style, we have to map them.
+        // Thankfully there is another handy attribute to achieve this: `BsonElement`.
+        // It takes the document field's name and maps it to the classes field name.
+        [BsonElement("title")] public String Title { get; set; }
+        [BsonElement("date")] public DateTime Date { get; set; }
```

Comment:
Not urgent, but would be cool to add a Lorem Ipsum "content" property later, and initialize it with Lorem Ipsum text
https://stackoverflow.com/questions/4286487/is-there-any-lorem-ipsum-generator-in-c

Reply:
👍